src/setOfPeople.js

```diff
@@ -38,7 +38,17 @@
  * @return {Set}
  */
 function setOfPeople(people) {
-  // write code here
+  const room = new Set();
+
+  people.forEach((elem, elemCopy) => {
```

Comment:
Don't use shortened words for variable names.
elem -> element (or even better - 'person', because you have set of people)